oracle/oracle.go

```diff
@@ -59,7 +59,7 @@ type Oracle interface {
 	GetExternalTimestamp(ctx context.Context) (uint64, error)
 	SetExternalTimestamp(ctx context.Context, ts uint64) error
 
-	GetMinTimestamp(ctx context.Context) (uint64, error)
+	GetAllTSOKeyspaceGroupMinTS(ctx context.Context) (uint64, error)
```

Comment:
Consider explain the detailed purpose and behavior of this API in comments somewhere. Here can be the place.